docs/source/pillars/computing_technology.md

```diff
@@ -176,79 +232,57 @@ _The ability of the TRE operator to administer and secure network infrastructure
+* - You must have a procedure for increasing/decreasing available resources.
+  - For cloud-based TREs this may involve scaling resources, such as virtual machines or databases, or deploying additional resources.
+    For on-premises TREs this may involve a procurement process to ensure that necessary resources are available.
+    Not all requests for capacity increase must necessarily be granted, but having a clear process will help projects understand when/why/how they can make use of additional capacity.
   - Mandatory
 ```
 
-### Capacity management
+### Billing
 
-_The ability of the TRE operator to ensure the right amount of resources are available at the right time to provide a service._
+The process of generating and managing invoices and bills for projects within the TRE.
+It involves calculation, issuance, and recording of payments and receipts.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-capacity-management
+:name: tab-infrastructure-billing
 
 * - Statement
   - Guidance
   - Importance
-* - You must ensure that all projects understand what resources are available and what the associated costs will be before the project starts.
-  - For on-premises systems this might be related to the available hardware, for cloud-based systems there might be limits on how many instances of a particular resource (_e.g._ GPUs) can be used. Projects should use this information to understand whether the available resources will be sufficient for their requirements.
-  - Mandatory
-* - You should ensure that the anticipated needs of projects can be satisfied using available resources.
-  - Note that this does not require you to accept requests for additional resources, but rather that promises made about resource availability before a project starts should be honoured wherever possible.
-  - Recommended
 * - You must ensure that the anticipated resource requirements will not result in overspending by the TRE.
   - For cloud-based TREs this may involve budgeting and/or restricting resource consumption on a project-by-project basis.
     For on-premises TREs this may involve managing expectations to match the available resource.
   - Mandatory
-* - You must have a procedure for increasing/decreasing available resources.
-  - For cloud-based TREs this may involve scaling resources, such as virtual machines or databases, or deploying additional resources.
-    For on-premises TREs this may involve a procurement process to ensure that necessary resources are available.
-  - Mandatory
-* - You must have a procedure to decide when to change capacity.
-  - Not all requests for capacity increase must necessarily be granted, but having a clear process will help projects understand when/why/how they can make use of additional capacity.
-  - Mandatory
 ```
 
 ### Configuration management
```

Comment:
This section feels in the wrong place. Shouldn't it be under (or alongside) infrastructure management?

Reply:
The idea here is that billing is part of managing capacity. The only statement we had about billing was to do with infrastructure, but we could/should add something about personpower here too. Would that make it more coherent for you @JimMadge ?

Reply:
One to raise with @machintim when he gets back.